focus-blocked/js/init.js

```diff
@@ -68,11 +85,17 @@ const isPageLoaded = Boolean(
 const isPageReloaded = Boolean(
   localStorage.getItem(LOCAL_STORAGE.IS_PAGE_RELOADED)
 );
-const isExternalHintRequired = Object.values(EXTERNAL_HINT_DOMAINS).includes(
-  domain
-);
 const version = urlParams.get('version');
 const show_tour = urlParams.get('show_tour') === 'true';
+const confirmSuperDistracting =
+  urlParams.get('confirm_super_distracting') === 'true';
+const confirmAIDistractingURL =
```

Comment:
@amanueltsfy have you implemented the different UI for the AI confirmation? (See the issue)

Reply:
The AI confirmation UI will feature the same interface, but with the addition of a modal that includes a subtitle.